### PR TITLE
[CI:DOCS] Release notes for v3.2.3

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+- Fixed a bug where `podman save` would refuse to save images with an architecture different from that of the host ([#10835](https://github.com/containers/podman/issues/10835)).
+- Fixed a bug where the `podman import` command did not correctly handle images without tags ([#10854](https://github.com/containers/podman/issues/10854)).
+- Fixed a bug where Podman's journald events backend would fail and prevent Podman from running when run on a host with systemd as PID1 but in an environment (e.g. a container) without systemd ([#10863](https://github.com/containers/podman/issues/10863)).
+- Fixed a bug where containers using rootless CNI networking would fail to start when the `dnsname` CNI plugin was in use and the host system's `/etc/resolv.conf` was a symlink ([#10855](https://github.com/containers/podman/issues/10855) and [#10929](https://github.com/containers/podman/issues/10929)).
+- Fixed a bug where containers using rootless CNI networking could fail to start due to a race in rootless CNI initialization ([#10930](https://github.com/containers/podman/issues/10930)).
+
+### Misc
+- Updated Buildah to v1.21.3
```
Comment (lsm5) [inline RELEASE_NOTES.md:13]: do we need to mention CVE ID? The CVE is public now.

Comment (lsm5) [reply RELEASE_NOTES.md:13]: @mheon  CVE-2021-3602

**Comment by TomSweeneyRedHat** [comment]
Other than @lsm5's question, LGTM

**Comment by mheon** [comment]
Re-pushed with note on the CVE

**Review by lsm5 — APPROVED** [review]
LGTM

**Pull request merged**